TESTS/mbedmicro-rtos-mbed/threads/main.cpp

```diff
@@ -174,6 +196,10 @@ void test_single_thread()
 template <int N, void (*F)(counter_t *)>
 void test_parallel_threads()
 {
+    char *dummy = new (std::nothrow) char[PARALLEL_THREAD_STACK_SIZE * N];
+    delete[] dummy;
+    TEST_SKIP_UNLESS_MESSAGE(dummy, "Not enough memory to run test");
```

Comment:
@davidsaada This doesnt seem right. We are trying to allocate the space and then deleting it right away, and then comparing it using the macro. It will always skip the test. Unless I am missing something here. Can you please check?

Reply:
The `delete[]` command has no effect on the value of `dummy` variable. The test will be skipped only if memory allocation fails. I could switch these two lines, but it won't change anything.